### DISCUSSION TITLE
Contact & authorization details for Fulfillment

### DISCUSSION THREAD

**Discussion by BLR-0118** [ONDC-Official/protocol-network-extension#63] [Announcements]
>Proposed changes for passing buyer contact details for fulfillment is [here](https://docs.google.com/document/d/1g077qKCWuLXdt-J8XuAw7AiGGXXjiPxDAcCyPzLcoSk/edit#). Includes enhancement for contacting buyer through IVR and PIN.

**Comment by BLR-0118** [comment]
@aakasjayaraj @mherle @VidyaGF @ravi-kumar-yadav @amitsaluja https://github.com/orgs/ONDC-Official/teams/participants

**Comment by ONDC-Tech** [comment]
Updated version of the doc is [here](https://docs.google.com/document/d/1g077qKCWuLXdt-J8XuAw7AiGGXXjiPxDAcCyPzLcoSk/edit#)
